tests/Admin/AdminTest.php

```diff
@@ -1731,6 +1732,35 @@ public function testGetSubject($id): void
         static::assertSame($model, $admin->getSubject()); // model manager must be used only once
     }
 
+    public function testSetSubject(): void
+    {
+        $this->expectNotToPerformAssertions();
+
+        $admin = new PostAdmin();
+        $admin->setModelClass(Post::class);
+        $admin->setModelManager($this->createMock(ModelManagerInterface::class));
```

Comment:
Not sure we need this call

Reply:
Not having this calls will give:
```
LogicException: Admin "Sonata\AdminBundle\Tests\Fixtures\Admin\PostAdmin" has no model manager.
```

Reply:
Oh no... that was before when `setSubject()` was using `getClass()` (which is using `getModelManager()`). Will remove it :+1: